### PR TITLE
Validate contact form

### PR DESCRIPTION
**Pull request by BenMukebo**
> ## Here are what was done to implement a client-side validation of the data in the contact form before submission:
> 
> - [ ] When the user submits the form, you check if the email is in lower case :
 > - If the validation is OK, the form is sent.
>  - If the validation is not OK, you show an error message to the user near the submit button
> - [ ] Use the valid js file to give style the the validation message. 
> - [ ] Update the README file,
> - [ ]  Fix all linter errors.

### CODE REVIEW

File: js/valid.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const form = document.getElementById('form');
+const email = document.getElementById('mail');
+const alertParent = document.querySelector('.btn-submit-alert');
+const small = document.createElement('small');
+alertParent.appendChild(small);
+small.innerText = 'Please enter a valid email address.';
+small.style.color = '#fff';
+small.style.backgroundColor = '#e74c3c';
+small.style.padding = '12px';
+small.style.visibility = 'hidden';
```
Comment (hemantso) [inline js/valid.js]: - [ ] As per milestone requirements, The error message fits the design from the chosen template in Figma (e.g., font, colours, images, tests, margins). Kindly try to follow the design pattern and change the font colour and background colour as they fit to design.

image

File: js/valid.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+form.addEventListener('submit', (e) => {
+  if (email.value !== email.value.toLowerCase()) {
+    small.style.visibility = 'visible';
+    email.style.border = '1px solid #e74c3c';
+    e.preventDefault();
```
Comment (hemantso) [inline js/valid.js]: - [ Optional ] We can make the design better by adding the set time out function which will disappear error message after giving information in a specified time.

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Kindly try to insert all images inside the images folder. It will help us to make our repo looks professional.

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/valid.js (hunk 1 above)
Comment (BenMukebo) [reply js/valid.js]: Hello hemant-soni-vst-au4 
 May you please check my changes and tell me if everything is okay?

22b

File: js/valid.js (hunk 2 above)
Comment (BenMukebo) [reply js/valid.js]: Thanks

**Review by DanteAlonsoHT — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @BenMukebo, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**